hooks/src/lib/utils/getCardNumberMaxLength.ts

```diff
@@ -0,0 +1,7 @@
+const getCardNumberMaxLength = (cardBrand: BrandType) => {
+  if (cardBrand === "Diners") return 14;
+  if (cardBrand === "AMEX") return 15;
+  return 16;
+};
```

Comment:
이런 정책 정보는 card brand 상수에 같이 정의해두어도 좋을 것 같아요. (지금도 좋습니다)

Reply:
카드 브랜드를 따로 상수로 정의했기에 매직넘버가 아닌 해당 상수로 처리하여도 충분히 가독성이 좋다 생각하여 아래와 같이 수정하겠습니다!

```suggestion
const getCardNumberMaxLength = (cardBrand: BrandType) => {
  if (cardBrand === CARD_BRAND.Diners) return 14;
  if (cardBrand === CARD_BRAND.AMEX) return 15;
  return 16;
};
```